### DISCUSSION TITLE
gmapping problem

### DISCUSSION THREAD

**Discussion by islemhm** [prl-mushr/mushr#34] [General]
hello ihave a problem when i run : rosrun gmapping slam_gmapping scan:=/scan and it show me this error message " [ WARN] [1623675167.536135530, 1039.105000000]: MessageFilter [target=odom ]: Dropped 100.00% of messages so far. Please turn the [ros.gmapping.message_filter] rosconsole logger to DEBUG for more information. " and it show me in rviz "no map received" any one can help me plzz i use ubuntu 18.04 and ros melodic

**Comment by MarkusSchiffer** [comment]
Hello islemhm, 
Have you looked at our newest tutorial? It's designed with Ubuntu 18.04 and ROS Melodic in mind. Give this a try: https://mushr.io/tutorials/mapping/

Feel free to reply if there are any problems with that tutorial!
- Markus

**Reply by islemhm** [reply]
@MarkusSchiffer thank you i will look at him
